components/textdiff/textdiff.js

```diff
@@ -43,7 +43,7 @@ var Type = function() {
   var textDiff = 'textdiff'
 
   this.text = ko.observable("Default");
-  this.value = ko.observable(textDiff);
+  this.value = ko.observable(ungit.config.diffType||textDiff);
```

Comment:
Sure - but I can't see any way to show a warning log. console.log is not displayed since it's a forked process - am I wrong?